src/benchmark/processes.py

```diff
@@ -3,6 +3,7 @@
 import platform
 import re
 from abc import ABC
+from pathlib import Path
```

Comment:
I think we agreed to use os or pathlib, not both at the same time

Reply:
Switched to os.path